utils/check-environment-variable.ts

```diff
@@ -0,0 +1,12 @@
+/*
+* Check if the environment variables are set
+* @param args - Environment variables to check
+* @throws Error if any of the environment variables is not set
+* @returns void
+* */
+export function checkEnvVariables(...args: string[]) {
+    const missingVariables = args.filter(arg => process.env[arg] === undefined);
+    if (missingVariables.length > 0) {
+        throw new Error(`The following environment variables are not set yet: ${missingVariables.join(', ')}. Please set them in .env file or pipeline environments.`);
```

Comment:
**Suggestion:** The `checkEnvVariables` function can be enhanced to log a warning for missing environment variables instead of throwing an error, allowing the application to continue running with default values.
> 🛰️  **Medium Priority** 
 > [category: **`enhancement`**, importance: **`5`**]
> **Reason:** Changing from an error to a warning for missing environment variables can make the application more resilient and user-friendly by not terminating the process immediately. However, this might lead to unnoticed configuration issues. 

```suggestion
    if (missingVariables.length > 0) {
        console.warn(`Warning: The following environment variables are not set yet: ${missingVariables.join(', ')}. Using default values.`);
    }
```